instagram/src/components/PostContainer/LikeSection.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import './Post.css';
+
+const LikeSection = props => {
+    let result = props.likes;
+    let myClassName = "likes";
+    if (props.likeStatus) {
+        result += 1;
+        myClassName = "liked";
+    }
+    return (
+        <div>
+            <div className={myClassName} onClick={props.incrementLike}>
+                <FontAwesomeIcon icon={['far', 'heart']} />
```

Comment:
Nice job figuring out how to increment/decrement a like and also changing the heart color